tests/test_application_form.py

```diff
@@ -247,28 +237,7 @@ def test_form_pages_contain_expected_fields(self):
                             )
                         assert found_input is not None, error_message
 
-    # def test_insecure_form_post_returns_403(self):
-    #     """
-    #     GIVEN Our FORM SERVICE Application is running
-    #     at FORMS_SERVICE_HOST and the given
-    #     FORMS_SERVICE_TEST_FORM has been created
-    #     WHEN testing the first given route in test_form_routes
-    #     THEN check that an insecure form post returns 403
-    #     """
-    #     route = test_form_pages[0]
-    #     path = route["path"]
-    #     route_rel = "/" + FORMS_SERVICE_TEST_FORM + path
-    #     url = FORMS_SERVICE_HOST + route_rel
-    #     kwargs = {}
-    #     # response = self.driver.requests(
-    #     #     method="POST",
-    #     #     url=url,
-    #     #     kwargs=kwargs)
-    #     data = parse.urlencode({"name": "Bad Data"}).encode()
-    #     req = request.Request(url, data=data)
-    #     response = request.urlopen(req)
-    #     assert response.code == 403
-
+    @pytest.mark.accesible
```

Comment:
*accessible

Reply:
Removed due to likelihood of sp's occurring